tests/perf-test/ReportActionCompose.perf-test.js

```diff
@@ -51,6 +50,22 @@ jest.mock('../../src/libs/actions/EmojiPickerAction', () => {
     };
 });
 
+jest.mock('../../src/components/withNavigationFocus', () => (Component) => {
+    function WithNavigationFocus(props) {
+        return (
+            <Component
+                // eslint-disable-next-line react/jsx-props-no-spreading
+                {...props}
+                isFocused={false}
+            />
+        );
+    }
+
+    WithNavigationFocus.displayName = 'WithNavigationFocus';
+
+    return WithNavigationFocus;
+});
```

Comment:
Why did we have to add this here now? I dont see related change that would require the navigationFocus?

Reply:
I had to add this to get rid of this warning from the console :

`Warning: Failed prop type: Invalid prop 'isFocused' of type 'object' supplied to 'AttachmentPickerWithMenuItems', expected 'boolean'. `

Changes requiring this mock have been added before.  Although the tests passed, the above warning appeared.